src/App.tsx

```diff
@@ -15,36 +17,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_BY_ALPH: number = 1;
+const SORT_BY_LENGTH: number = 2;
+
+const sortGoods = (goods: string[], key: number): string[] => {
+  switch (key) {
+    case SORT_BY_ALPH:
+      return goods.sort((a, b) => a.localeCompare(b));
+    case SORT_BY_LENGTH:
+      return goods.sort((a, b) => a.length - b.length);
+    default:
+      return goods;
+  }
+};
```

Comment:
[CODE STYLE] - Function's name should start with a verb. It's better to name your function `sortGoodsByKey` instead of `sortGoods` to make it clear what the function does.